with-descope/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "expo": "~49.0.15",
```

Comment:
We recently released [SDK 50](https://expo.dev/changelog/2024/01-18-sdk-50), can you please upgrade it to that?